lib/screen/settings/settings_page.dart

```diff
@@ -326,8 +326,9 @@ class _SettingsPageState extends State<SettingsPage>
             }),
       const SizedBox(height: 10),
       StatefulBuilder(builder: (BuildContext context, StateSetter setState) {
-        final isLatestVersion = _versionCheck?.storeVersion
```

Comment:
using string compareTo may be not correct here. eg: 0.9.0 > 0.10.0
We had a util `compareVersion` in `helpers.dart` already I think.